.clang-format

```diff
@@ -0,0 +1,60 @@
+BasedOnStyle: LLVM
```

Comment:
Great!
Is it fast to do clang analysis?

Please propose on how we reinforce this during development.